optuna_dashboard/_app.py

```diff
@@ -176,6 +177,7 @@ def delete_study(study_id: int) -> dict[str, Any]:
             delete_all_artifacts(artifact_store, storage, study_id)
 
         try:
+            note.delete_study_notes(storage, study_id)
```

Comment:
`study_system_attrs` will be removed in `storage.delete_study()` so that you don't need to delete notes here.

```suggestion
```